src/modules/delete_todo.js

```diff
@@ -2,8 +2,6 @@ import getTodos from './get_todo.js';
 import AddTodo from './add_todo.js';
 // import updateTodo from './update_todo.js';
 
-// const todoElement = document.querySelector('.todoList');
-
```

Comment:
# Required Changes
I think its not nice to add comments in your code, remove unused comments